package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@floip/flow-runner",
-  "version": "1.0.0-rc2.5",
+  "version": "1.0.0-rc2.6",
```

Comment:
Let's rely on `yarn version` update this field as outlined in previous PR note 😻

Reply:
Sure, I'll revert this and will rely on previous `yarn version` note <3